python-avd/pyavd/_eos_designs/structured_config/underlay/utils.py

```diff
@@ -270,8 +269,8 @@ def _get_l2_as_subint(
             eos_cli=svi.raw_eos_cli,
         )
 
-        if (flow_tracking := link.get("flow_tracking")) and (
-            flow_tracker := self.shared_utils.new_get_flow_tracker(flow_tracking, EosCliConfigGen.EthernetInterfacesItem.FlowTracker)
+        if link.flow_tracking and (
+            flow_tracker := self.shared_utils.new_get_flow_tracker(link.flow_tracking, EosCliConfigGen.EthernetInterfacesItem.FlowTracker)
```

Comment:
we don't need the same check in the port_channel_interfaces.py module as the called function handles this gracefully
```suggestion
        if flow_tracker := self.shared_utils.new_get_flow_tracker(link.flow_tracking, EosCliConfigGen.EthernetInterfacesItem.FlowTracker:
```